kpatch-build/kpatch-build

```diff
@@ -459,10 +481,15 @@ grep -q "CONFIG_DEBUG_KERNEL=y" "$OBJDIR/.config" || die "kernel doesn't have 'C
 if grep "CONFIG_LIVEPATCH=y" "$OBJDIR/.config" > /dev/null; then
 	# The kernel supports livepatch.
 	KBUILD_EXTRA_SYMBOLS=""
+	if version_gte ${ARCHVERSION//-*/} 4.7.0; then
+		USE_KLPRELAS=true
+		KPATCH_LDFLAGS="--unique=.parainstructions --unique=.altinstructions"
```

Comment:
Ah, when I made this comment, I think I had forgotten that create-klp-module runs on the .ko rather than the individual .o files.  Is there any reason why it couldn't be renamed to `create-klp-object` and run on the .o file instead?